web/packages/teleport/src/Integrations/Enroll/IntegrationTiles.tsx

```diff
@@ -38,7 +38,8 @@ export function IntegrationTiles({
   hasIntegrationAccess?: boolean;
   hasExternalAuditStorage?: boolean;
 }) {
-  const externalAuditStorageEnabled = cfg.externalAuditStorage;
+  const externalAuditStorageEnabled =
+    cfg.entitlements.ExternalAuditStorage.enabled;
```

Comment:
@mcbattirola just wanted to double check this change is equivalent and is ok

Reply:
should `cfg.externalAuditStorage` then be marked deprecated??

Reply:
Yes, `cfg.externalAuditStorage` is the old way, we should use the entitlements now.

Here's where we set `cfg.externalAuditStorage` based on the entitlement value for compatibility: https://github.com/gravitational/teleport/blob/ee1726ef684b7162de241ba63674cfdcdbfe074b/lib/web/apiserver.go#L1872


> should cfg.externalAuditStorage then be marked deprecated??

Yes. Do you want me to open a PR, or you prefer including it in this one?

Reply:
i'll do it in another PR